llvm/docs/ReleaseNotes.rst

```diff
@@ -201,6 +201,14 @@ Changes to the C API
 Changes to the CodeGen infrastructure
 -------------------------------------
 
+* A new debug type ``isel-dump`` is added to show only the SelectionDAG dumps
+  after each ISel phase (i.e. ``-debug-onlu=isel-dump``). This new debug type
+  can be filtered by function names using ``-filter-print-funcs=<function names>``,
+  the same flag used to filter IR dumps after each Pass. Note that to be
+  compatible with the existing ``-debug-only=isel``, the latter will still
+  print SelectionDAG dumps of every single functions regardless of
```

Comment:
I think this sentence is a bit confusing to read. For example, I'm not sure what `the latter` refers to in this context.

Do you think this change would still make sense?

```
Note that -debug-only=isel will take precedence over the new behavior and print SelectionDAG dumps ...
```

Reply:
Yes, that's better. It's fixed now.